src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  gap: 4px;
```

Comment:
Don't use gap yet, the test does not support this parameter yet. 
It is a very useful thing, and you will use it in your landings.
![image](https://user-images.githubusercontent.com/57747481/173520311-caa046d3-3485-4435-9c7d-d245a1c14904.png)